### PR TITLE
Fix issue with steps not running in workflow dispatch

### PR DESCRIPTION
**Pull request by calebkiage**

### CODE REVIEW

**Review by andrueastman — APPROVED** [review]

File: .github/workflows/docker-publish.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
           registry: docker.pkg.github.com
       - name: Push to GitHub Packages - Nightly
-        if: contains(github.ref, 'refs/head/main')
+        if: contains(github.ref, 'refs/head/main') || github.event.inputs.tag == ''
```
Comment (baywet) [inline .github/workflows/docker-publish.yml]: this is effectively nullifying the first condition and will make this workflow run for any branch, which I don't think is what we want.

**Review by baywet — CHANGES_REQUESTED** [review]
see prior comment

File: .github/workflows/docker-publish.yml (hunk 1 above)
Comment (calebkiage) [reply .github/workflows/docker-publish.yml]: Fixed

File: .github/workflows/docker-publish.yml [hunk 2]
```diff
@@ -43,7 +43,7 @@ jobs:
             "user=${{ secrets.NUGET_USER }}"
             "token=${{ secrets.NUGET_PASSWORD }}"
       - name: Push to GitHub Packages - Release
-        if: contains(github.ref, 'refs/tags/v')
+        if: contains(github.ref, 'refs/tags/v') || startsWith(github.event.inputs.tag, 'v')
```
Comment (baywet) [inline .github/workflows/docker-publish.yml]:
```suggestion
        if: contains(github.ref, 'refs/tags/v')
```

**Review by baywet — CHANGES_REQUESTED** [review]

**Review by baywet — APPROVED** [review]

**Pull request merged**